device-types/Opengear/ACM7008-2-L.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Opengear
+model: ACM7008-2-L
+slug: acm7008-2-l
+part_number: ACM7008-2-L
+u_height: 1
+is_full_depth: false
+comments: ACM7008 Console Server with 8 Console Ports on the rear of the device.
+console-server-ports:
```

Comment:
Is [this image](https://webobjects2.cdw.com/is/image/CDW/5651010d?$product-detail$) the right model? 

If so, it mgiht be better to lavel them exactly as they appear/show in the config